tests/test_pytest_plugin.py

```diff
@@ -725,3 +725,38 @@ def test_a():
 """
         ),
     )
+
+
+
+def test_snapshot_dir_config(project):
+    project.pyproject(
+        """
+[tool.inline-snapshot]
+snapshot-dir = "tests/snapshots"
+"""
+    )
+
+    project.setup(
+        """
+from inline_snapshot import outsource, snapshot
+
+def test_outsource():
+    assert outsource("hello", suffix=".html") == snapshot()
+"""
+    )
+
+    result = project.run("--inline-snapshot=create")
+    assert result.ret == 0
+    assert project.source == snapshot("""\
+from inline_snapshot import outsource, snapshot
+
+from inline_snapshot import external
+
+def test_outsource():
+    assert outsource("hello", suffix=".html") == snapshot(external("2cf24dba5fb0*.html"))
+""")
+
+    project.run("--inline-snapshot=fix")
+    assert result.ret == 0
+
+    assert project.storage("tests/snapshots")
```

Comment:
Maybe I could actually check the value here.

Reply:
``` python
assert project.storage("tests/snapshots") == snapshot()
```